Src/Support/Google.Apis.Core/Util/ExponentialBackOff.cs

```diff
@@ -28,48 +28,45 @@ public class ExponentialBackOff : IBackOff
         /// <summary>The maximum allowed number of retries.</summary>
         private const int MaxAllowedNumRetries = 20;
 
-        private readonly TimeSpan deltaBackOff;
+        /// <summary>The random instance which generates a random number to add the to next back-off.</summary>
+        private static readonly Random Random = new Random();
```

Comment:
Random isn't thread-safe - at least in some implementations. (I think it may be in .NET 8+but that's only a vague memory.)

We should probably lock on every use of it.

Reply:
(I've just noticed that this isn't actually introducing a new Random, just moving it - but the comment still stands... we should probably fix that in a separate PR.)